### PR TITLE
[5.3] Fix pcntl extension error for queue

### LINKED ISSUE #14909: [5.3]PHP have to --enable-pcntl if use queue
I tried this under my windows with PHP 7.0 (no way to enable pcntl):
`php artisan queue:work  --timeout=360 --queue=high,low --sleep=3 --tries=3`
Will get this exception.
```
  [RuntimeException]
  The pcntl extension is required in order to specify job timeouts.
```
But same project in my server ( Ubuntu 16.04 with PHP 7.0 enable pcntl), it works fine.
So if it is necessary to add in "Server Requirements" ?

### PR DESCRIPTION
**Pull request by srmklive**
Fixes #14909

### CODE REVIEW

File: src/Illuminate/Queue/Console/ListenCommand.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
 use Illuminate\Console\Command;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Input\InputArgument;
+use RuntimeException;
```
Comment (GrahamCampbell) [inline src/Illuminate/Queue/Console/ListenCommand.php]: Please sort by length.

**Comment by GrahamCampbell** [comment]
Thanks, but you forgot to add a throws annotation. :)

**Comment by srmklive** [comment]
@GrahamCampbell You are right :). Fixing it.

File: src/Illuminate/Queue/Console/ListenCommand.php [hunk 2]
```diff
@@ -47,6 +48,7 @@ public function __construct(Listener $listener)
      * Execute the console command.
      *
      * @return void
+     * @throws RuntimeException
```
Comment (GrahamCampbell) [inline src/Illuminate/Queue/Console/ListenCommand.php]: Missing newline and slash.

**Pull request merged**